lib/scarpe/wv/button.rb

```diff
@@ -24,9 +24,13 @@ def element
 
     def style
       styles = {}
-
+      styles["background-color"] = @color
```

Comment:
Yeah... At some point we really need to go through Shoes and figure out what properties/styles we're supporting.

In the mean time, this is fine.